ovsctl/ovsctl.go

```diff
@@ -55,7 +57,20 @@ type Ovsctl struct {
 }
 
 func NewOvsctl() Ovsctl {
-	return Ovsctl{execcli: platform.NewExecClient()}
+	execcli := platform.NewExecClient()
+	// try to ensure ovs-vswitchd is active (for Ubuntu 18), otherwise log failure to start the daemon
+	if output, err := execcli.ExecuteCommand("lsb_release -is"); err == nil && strings.Contains(output, "Ubuntu") {
```

Comment:
`lsb_release` may not be installed. is there a better way to do this than exec and `string.Contains`?

Reply:
parse /etc/os-release?

Reply:
/etc/os-release is better